pkg/controller/component/component_controller.go

```diff
@@ -282,3 +299,56 @@ func generateBuildConfig(namespace string, name string, builder *builderImage, g
 		},
 	}
 }
+
+func generateDeploymentConfig(namespace string, name string) *v1.DeploymentConfig {
+	labels := map[string]string{
+		"app": name,
+	}
+	return &v1.DeploymentConfig{
```

Comment:
How about creating a Deployment instead of a DeploymentConfig? I'm saying this because DeploymentConfigs don't exist in Kubernetes.

Reply:
agreed, prefix is useless, label selectors should be used instead. I'll remove.

Reply:
Cool, thank you.